protocols/virtual-fund/virtual-fund.go

```diff
@@ -285,16 +297,14 @@ func (s VirtualFundObjective) fundingComplete() bool {
 	// Each peer commits to an update in L_{i-1} and L_i including the guarantees G_{i-1} and {G_i} respectively, and deducting b_0 from L_{I-1} and a_0 from L_i.
 	// A = P_0 and B=P_n+1 are special cases. A only does the guarantee for L_0 (deducting a0), and B only foes the guarantee for L_n (deducting b0).
 
-	n := s.n
-
 	switch {
-	case s.MyRole == 0: // Alice
+	case s.MyRole == Alice:
 		return s.ToMyRight.ledgerChannelAffordsExpectedGuarantees()
-	case s.MyRole < n+1: // Intermediary
-		return s.ToMyRight.ledgerChannelAffordsExpectedGuarantees() && s.ToMyLeft.ledgerChannelAffordsExpectedGuarantees()
-	case s.MyRole == n+1: // Bob
+	case s.MyRole == Bob:
 		return s.ToMyLeft.ledgerChannelAffordsExpectedGuarantees()
-	default: // Invalid
+	case s.MyRole > I1:
```

Comment:
```suggestion
	case s.MyRole >= I1:
```
Or `> Bob:`, but probably the above is better.

Reply:
Is this also indicating a gap in our test suite?